spec/factories/devise.rb

```diff
@@ -12,11 +12,11 @@
       slug {"slug"}
     end
 
-    factory :completed_lesson do
-      lesson_id {1}
-      user_id {1}
-      completed {true}
-    end
+    # factory :completed_lesson do
+    #   lesson_id {1}
+    #   user_id {1}
+    #   completed {true}
+    # end
```

Comment:
You can delete this code, as the `completed_lesson` factory was moved to another file

Reply:
I initially kept this because it was not my own code. However, I removed it now with a commit that explains why.